library/ix-dev/community/immich/templates/_postgres.tpl

```diff
@@ -1,7 +1,23 @@
 {{- define "postgres.workload" -}}
+  {{- if not .Values.pgvectorImage -}}
+    {{/* Only to make CI upgrade happy,
+      can be removed on next version bump */}}
+    {{- $_ := set .Values "pgvectorImage" (dict
+      "repository" "tensorchord/pgvecto-rs"
+      "pullPolicy" "IfNotPresent"
+      "tag" "pg15-v0.1.11"
+    ) -}}
+  {{- end }}
+
+{{- $psql := "PGPASSWORD=${POSTGRES_PASSWORD} psql --host=${POSTGRES_HOST} --dbname=${POSTGRES_DB} --username=${POSTGRES_USER}" -}}
+{{- $tasks := (list
+  (printf "%s -c \"ALTER DATABASE ${POSTGRES_DB} REFRESH COLLATION VERSION;\"" $psql)
```

Comment:
this is needed due the slight downgrade from pg 15.2 to pg 15.0 (the only image/version that pgvector has available on the same major version).

Thankfully its not very expensive (on my testing it was instant) and its does not fail or re-run if there isn't something to refresh